internal/server/ptypes/workspace.go

```diff
@@ -0,0 +1,37 @@
+package ptypes
+
+import (
+	"github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/imdario/mergo"
+	"github.com/mitchellh/go-testing-interface"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/waypoint/internal/pkg/validationext"
+	pb "github.com/hashicorp/waypoint/internal/server/gen"
+)
+
+func TestGetWorkspaceRequest(t testing.T, src *pb.GetWorkspaceRequest) *pb.GetWorkspaceRequest {
```

Comment:
Should this func go in `workspace_test.go`?

Reply:
Nah part of our public API is exposing helpers to get valid values, i.e. this one and `TestJob` and so on.